docs/quickstart.asciidoc

```diff
@@ -35,13 +35,20 @@ NOTE: If you are using Amazon EKS, make sure the Kubernetes control plane is all
 
 . Install link:https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/[custom resource definitions] and the operator with its RBAC rules:
 +
+If you are using a Kubernetes cluster in version 1.12 or below, use:
++
+[source,sh,subs="attributes"]
+----
+kubectl apply -f https://download.elastic.co/downloads/eck/{eck_version}/all-in-one-no-validation.yaml
+----
++
+If you are using a Kubernetes cluster from version 1.13, use:
++
 [source,sh,subs="attributes"]
 ----
 kubectl apply -f https://download.elastic.co/downloads/eck/{eck_version}/all-in-one.yaml
 ----
```

Comment:
I would invert the order so that the most common version is the first one.

```suggestion
For Kubernetes clusters running version 1.13 or higher:

[source,sh,subs="attributes"]
----
kubectl apply -f https://download.elastic.co/downloads/eck/{eck_version}/all-in-one.yaml
----

For Kubernetes clusters running version 1.12 or lower:

[source,sh,subs="attributes"]
----
kubectl apply -f https://download.elastic.co/downloads/eck/{eck_version}/all-in-one-no-validation.yaml
----
````

Reply:
Absolutely.